README.md

```diff
@@ -0,0 +1,205 @@
+# NGSA Loderunner - A web request validation tool
+
+Loderunner (L8s) is an internal web request validation tool that we use to run end-to-end tests and long-running smoke tests.
+
+## Quick Start
+
+L8s can be run as a docker container, since it is available at gitcr.io, most of its features and functionality are inherited from its parent code base [WebV](https://github.com/microsoft/webvalidate).
+
+## Running as a docker container
+
+```bash
+
+# pull image from GitHub Container Repository 
+docker pull ghcr.io/retaildevcrews/ngsa-lr:beta
+
+```
+
+Experiment with L8s
+
+```bash
+# run L8s with --help option; this should output command line options shown below
```

Comment:
You could also add additional markdown to direct someone to "below"

I think it's something like
[below](#command-line-parameter)

Reply:
Fixed